src/modules/func.js

```diff
@@ -34,19 +35,19 @@ displayTask();
 
 const updateIndex = () => {
   for (let k = 0; k < tasks.length; k += 1) {
-    tasks[k].index = 1;
+    tasks[k].index = k;
```

Comment:
- [ ] New entries **do not match** the property index. The _property index_ must be set to the value of the _new array length_ (i.e. if a 5th task is added to the list, the index of that task should equal 5), as requested in the previous review.

**Index mismatch in your application:**
![image](https://user-images.githubusercontent.com/12745474/160237322-8dd84ddb-09dd-442e-8274-d15e902f425f.png)